horace_core/sqw/@sqw/sqw.m

```diff
@@ -429,6 +432,14 @@ function  save_xye(obj,varargin)
             obj.main_header.creation_date = val;
             obj.data.creation_date = val;
         end
+        %
+        function is = get.is_filebacked(obj)
+            if obj.has_pixels
+                is = obj.pix.is_filebacked;
+            else
+                is = true;
```

Comment:
I'm surprised by this. If it has no pixels, then nothing will be stored to file and so it should work in memory. Please comment.

Reply:
yes, this is incorrect. I've already changed this 440 should be `false`